Button/button.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bricks component library</title>
+    <link rel="stylesheet" href="/css/style.css">
+    <link rel="stylesheet" href="button.css">
+</head>
+<script src="https://kit.fontawesome.com/26b5866028.js" crossorigin="anonymous"></script>
+<body>
+    <div class="wrapper">
```

Comment:
It's good to wrap the main nav in a "header" tag.

Reply:
Done with these changes.